File: javascript/difference.js

difference = arr => {
	var arr2 = []
	for (let i = 0;i < arr.length; i++){
		arr2 = arr2.concat(arr[i])
	}

	result = {}
	for(let i = 0; i<arr2.length; i++){
		if (!(arr2[i] in result)){
			result[arr2[i]] = 1
		}else{
			result[arr2[i]]++
		}
	}

	result2 = []
	for(let i in result){
		if (result[i] ===1){
			result2.push(i)
		}
	}
	console.log(result2)
}

arr = [[1,2,3],[2,3,4,9],[2,4,5]]
difference(arr)

File: python/difference.py

def differencce(arr):
	arr2 = []
	for i in arr:
		arr2 += i

	result1 = {}
	for i in arr2:
		if i not in result1:
			result1[i] = 1
		else:
			result1[i] += 1

	result = []
	for i in result1:
		if result1[i] == 1:
			result.append(i)
	print(result)

arr= [[1,2,3],[2,3,4,9],[2,3,4,5]]
differencce(arr)